src/main/kotlin/app/revanced/patches/youtube/layout/shorts/button/fingerprints/PivotBarButtonTabEnumFingerprint.kt

```diff
@@ -22,9 +22,6 @@ object PivotBarButtonTabEnumFingerprint : MethodFingerprint(
     AccessFlags.PUBLIC or AccessFlags.FINAL,
     listOf("Z"),
     listOf(
-        Opcode.CHECK_CAST,
```

Comment:
Is the pattern still gonna work for the versions listed in `ShortsButtonCompatibility`? If you are not sure, remove all versions you have not tested this pattern.

Reply:
> Is the pattern still gonna work for the versions listed in `ShortsButtonCompatibility`? If you are not sure, remove all versions you have not tested this pattern.

As per your advice I've tested it on all versions marked in ShortsButtonCompatibility
: 17.14.35, 17.17.34, 17.19.36, 17.20.37, 17.22.36, 17.23.35, 17.23.36, 17.24.34, 17.24.35, 17.25.34, 17.26.35, 17.27.39, 17.28 .34, 17.29.34, 17.32.35
The 'hide-shorts-button' and 'disable-create-button' patches worked fine in all versions marked in ShortsButtonCompatibility.

Similarly, I have also tested all versions marked in WideSearchbarCompatibility.
: 17.25.34, 17.26.35, 17.27.39, 17.28.34, 17.29.34, 17.32.35
The 'enable-wide-searchbar' patch worked fine in all versions indicated in WideSearchbarCompatibility.